lib/index.js

```diff
@@ -48,14 +48,27 @@ function getRepo(options) {
  * Push a git branch to a remote (pushes gh-pages by default).
  * @param {string} basePath The base path.
  * @param {Object} config Publish options.
- * @param {Function} done Callback.
+ * @param {Function} callback Callback.
  */
-exports.publish = function publish(basePath, config, done) {
+ /*eslint-disable no-console */
+exports.publish = function publish(basePath, config, callback) {
   if (typeof config === 'function') {
-    done = config;
+    callback = config;
     config = {};
   }
 
+  var done = function(arguments) {
+    try {
+      (callback || function(err) {
+        if (!err) {
+          console.log(err)
```

Comment:
I'm just confused why you would want to log `undefined` or other falsy values.

To be clear, did you mean to write this:
```js
if (err) {
  console.log(err);
}
```
instead of this:
```js
if (!err) {
  console.log(err);
}
```
?